archive/p/php/quick_sort.php

```diff
@@ -0,0 +1,26 @@
+<?php
```

Comment:
Can you rename this using a hyphen? That's the naming convention we've been using, and your code won't be tested otherwise.